sysutils/ntpstat/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           Makefile 1.0
+
+github.setup        darkhelmet ntpstat 86e0a68df62e7f6b6c5ee97ee9d15b5e28a4b359
+version             0.2
+categories          sysutils net
+platforms           darwin
+license             GPL-2
+maintainers         {hotmail.com:amtor @RobK88} \
+                    openmaintainer
+
+description         ${name} shows the network time protocol (ntp) status
+
+long_description    ${description}. \
+                    ${name}  will  report the synchronisation state of the \
+                    NTP daemon running on the local machine.  If the local \
+                    system is found to be synchronised to a reference time \
+                    source, ${name} will report the approximate time accuracy.
+
+homepage            https://web.archive.org/web/20121207230716/http://people.redhat.com/rkeech/#ntpstat
+
+github.tarball_from archive
```

Comment:
can you please move this right under the `github.setup` line (I know it works either way, but I'd prefer to keep the `github` related stuff close together)